src/DataList/DataList.js

```diff
@@ -462,6 +462,19 @@ var DataList = module.exports = kind(
 			sup.apply(this, arguments);
 		};
 	}),
+
+	/**
+	* @private
+	*/
+	teardownRender: kind.inherit(function (sup) {
```

Comment:
@ryanjduffy, did you consider implementing the (relatively new) `beforeTeardown()` lifecycle method, rather than overriding `teardownRender()`?